zerologr.go

```diff
@@ -84,14 +88,13 @@ func (ls *LogSink) Init(ri logr.RuntimeInfo) {
 
 // Enabled tests whether this LogSink is enabled at the specified V-level.
 func (ls *LogSink) Enabled(level int) bool {
-	// Optimization: Info() will check level internally.
-	const traceLevel = 1 - int(zerolog.TraceLevel)
-	return level <= traceLevel
+	return ls.l.GetLevel() <= zerologLevel(level)
```

Comment:
A unit test for this behavior would be good.

Reply:
I borrowed the tests from logrusr and adapted them for zerologr, then added tests for Enabled() and zerologLevel().  zerologr actually passed all of the logrusr tests (except the ones I commented) without much adaptation to the tests.  I'll let you decide if you like to keep them.  Otherwise, only the Enabled() test is what you asked for, so the file be stripped down to just that one.  The logrusr tests are a decent set, though.